app/src/main/java/com/duycuong/weather/utils/BindingUtils.java

```diff
@@ -4,10 +4,15 @@
 import android.support.design.widget.TabLayout;
 import android.support.v4.view.PagerAdapter;
 import android.support.v4.view.ViewPager;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.RecyclerView;
 import android.widget.ImageView;
 import android.widget.TextView;
+
 import java.math.RoundingMode;
 import java.text.DecimalFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
```

Comment:
ERROR: Unused import: java.util.Locale. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports